conjur/credentials_from_file.py

```diff
@@ -0,0 +1,69 @@
+# -*- coding: utf-8 -*-
+
+"""
+CredentialsFromFile module
+
+This module holds the logic for writing user-specific credentials
+to a netrc file on the user's machine when they login
+"""
+
+# Builtins
+import logging
+import netrc
+import os
+import stat
+
+# Internals
+from conjur.constants import DEFAULT_NETRC_FILE
+
+class CredentialsFromFile:
+    """
+    CredentialsFromFile
+
+    This class holds logic when credentials are kept in the netrc
+    """
+    @classmethod
+    def save(cls, credential_data):
+        """
+        Method that writes user data to a netrc file
+        and updates permissions on the file
+        """
+        with open(DEFAULT_NETRC_FILE, 'w') as netrc_file:
+            # pylint: disable=line-too-long
+            netrc_file.write(f"machine {credential_data.machine}\nlogin {credential_data.login}\npassword {credential_data.password}\n")
+            netrc_file.close()
+
+        # Ensures that the netrc file is only available to owner
+        os.chmod(DEFAULT_NETRC_FILE, stat.S_IRWXU)
+
+    @classmethod
+    def remove(cls):
+        """
+        Method that removes netrc data from the file.
+        Triggered during a logout
+        """
+        if os.path.exists(DEFAULT_NETRC_FILE) and os.path.getsize(DEFAULT_NETRC_FILE) != 0:
+            open(DEFAULT_NETRC_FILE, "w").close()
+        else:
+            raise Exception("Please log in")
+
+    @classmethod
+    def load(cls):
+        """
+        Method that loads the netrc data.
+        Triggered before each CLI action
+        """
+        # pylint: disable=logging-fstring-interpolation
+        logging.info(f"Using netrc as credential store {DEFAULT_NETRC_FILE}")
```

Comment:
since you don't use credentials store, this log line is more misleading than informative.
Better:
```suggestion
        logging.info(f"Using {DEFAULT_NETRC_FILE} file to save credential")
```

Reply:
👌